compiler/src/dotty/tools/dotc/Run.scala

```diff
@@ -398,3 +405,12 @@ class Run(comp: Compiler, ictx: Context) extends ImplicitRunInfo with Constraint
   given runContext[Dummy_so_its_a_def]: Context = myCtx.nn
   assert(runContext.runId <= Periods.MaxPossibleRunId)
 }
+
+object Run {
+  extension (run: Run | Null)
```

Comment:
In fact, that will not worked ExplicitNulls is turned off. Or rather, it will be unsafe since then you can fail with an NPE. So I think it's better not to use the same names in the extension and in class Run.

Reply:
Ah, I see.  OK, I'll mangle the names.